mungegithub/submit-queue/www/index.html

```diff
@@ -148,9 +148,12 @@ <h2 class="md-toolbar-tools">Google End-to-End Results</h2>
                 <md-chips ng-model="cntl.builds" readonly="true">
                   <md-chip-template title="{{$chip.msg}}">
                     <span style="color: {{$chip.color}}">{{$chip.state}}</span> <a ng-href="https://storage.cloud.google.com/kubernetes-jenkins/logs/{{$chip.name}}/{{$chip.id}}/">{{$chip.name}}</a>
-                    <span style="color: {{$chip.color}}">{{$chip.msg}}</span>
+                    <span style="color: {{$chip.color}}">{{$chip.msg}}</span> <span title="Job stability">{{$chip.stability}}</span>
                   </md-chip-template>
                 </md-chips>
+                <br>
+                <h2 class="md-title">Overall Health: <span id="e2e-health"></span></h2>
```

Comment:
I'm not certain but I think you can replace the `<span></span>` with just `{{ cntl.OverAllHealth }}` and then set `self.OverAllHealth` everytime you update it...